pkg/common/config/config.go

```diff
@@ -50,10 +49,12 @@ const (
 	EnvGCConfig = "GC_CONFIG"
 	// DefaultpvCSIProviderPath is the default path of pvCSI provider config
 	DefaultpvCSIProviderPath = "/etc/cloud/pvcsi-provider"
-	// DefaultFSSConfigMapName is the default name Feature states config map
-	DefaultFSSConfigMapName = "csi-feature-states"
-	// DefaultFSSConfigMapNamespaceVanillaK8s is the default value for Feature state config map namespace
-	DefaultFSSConfigMapNamespaceVanillaK8s = "kube-system"
+	// DefaultSupervisorFSSConfigMapName is the default name of Feature states config map in Supervisor cluster
```

Comment:
Can you also add a note that this is replicated by the system onto any TKG's deployed?

Reply:
Done